docs/pages/branding/careers.tsx

```diff
@@ -0,0 +1,491 @@
+                {'Perks & benefits'}
+              </Typography>
+              <Typography color="text.secondary" sx={{ mb: 2 }}>
+                To help you go above and beyond with us, we provide:
+              </Typography>
+              {[
+                ['Remote work:', 'Our entire company is distributed.'],
+                [
+                  'Gatherings:',
+                  'We meet up once or twice a year for a short week of meetings, events, and fun!',
+                ],
+                [
+                  'Equipment:',
+                  'MUI will let you choose new hardware of your choice (up to $2,500 USD).',
+                ],
+                ['Time off:', 'We provide five weeks of paid time off.'],
+              ].map((textArray) => (
+                <Box key={textArray[0]} sx={{ display: 'flex', alignItems: 'flex-start', mt: 1 }}>
+                  <IconImage name="yes" />
+                  <Typography variant="body2" color="text.primary" sx={{ ml: 1 }}>
+                    <span style={{ fontWeight: 600 }}>{`${textArray[0]}  `}</span>
+                    {textArray[1]}
+                  </Typography>
+                </Box>
+              ))}
+            </Grid>
+            <Grid item xs={6} md={3}>
+              <Widget
+                title="Handbook"
+                url={'https://www.notion.so/mui-org/Handbook-f086d47e10794d5e839aef9dc67f324b'}
+              >
+                <Typography variant="body2" color="text.secondary" sx={{ mb: 1 }}>
+                  See how we run the company and the way we work.
+                </Typography>
+              </Widget>
+            </Grid>
+            <Grid item xs={6} md={3}>
+              <Widget title="Blog" url={'https://medium.com/material-ui'}>
+                <Typography variant="body2" color="text.secondary" sx={{ mb: 1 }}>
+                  Check behind the scenes and news from the company.
+                </Typography>
+              </Widget>
+            </Grid>
+          </Grid>
+        </Container>
+      </Box>
+      {/* Open roles */}
+      <Container sx={{ py: { xs: 4, md: 8 } }}>
+        <Box sx={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
+          <div>
+            <Typography variant="h2" sx={{ my: 1 }}>
+              Open roles
+            </Typography>
+            <Typography color="text.secondary" sx={{ mb: 2, maxWidth: 450 }}>
+              The company is bootstrapped (so far). It was incorporated in mid-2019 and yet growing
+              fast (x2-3 YoY). We doubled the team in 2020 and are on track to do the same in 2021.
+              We&apos;re looking for help keep growing in the following areas:
+            </Typography>
+          </div>
+          <div style={{ display: 'flex', alignSelf: 'start', paddingTop: '0.5em' }}>
+            <Typography fontWeight="bold">All areas</Typography>
+            <ArrowDropDown color="primary" fontSize="small" sx={{ ml: '1em' }} />
```

Comment:
About the "_All areas_" button: the idea is for it to be a select that filters the open positions per department. See the [Doist careers page](https://doist.com/careers), the one I used for reference, and you'll quickly get it!

Reply:
@siriwatknp I'll be resolving your question about this, answering it here 😬

Reply:
Got ya. I think it make sense if we have a lot of open roles. It is even worst when there are few open roles like
<img width="1177" alt="Screen Shot 2564-09-07 at 00 17 20" src="https://user-images.githubusercontent.com/18292247/132248417-d5025651-b032-4152-8622-e8ed853335a8.png">

I vote to remove it at this point.

Reply:
Ok for me, we can leave it out for now, once MUI is with 500 open positions and 2k contributors, we roll it out 😬

Reply:
Okay, I removed the all areas button for now!